pkger/parser.go

```diff
@@ -144,7 +151,7 @@ func parseSource(r io.Reader, opts ...ValidateOptFn) (*Pkg, error) {
 		b = bb
 	}
 
-	contentType := http.DetectContentType(b)
+	contentType := http.DetectContentType(b[:512])
```

Comment:
quick question, i remembered it used to like b[:512], is there any reason to change it back and forth

Reply:
I messed up. I dorked it up, but this is back to what it should be. It should only ever check the first 512 bytes. Although, don't think it hurts to give it the entire byte array, just not common to see it that way.